tag-tmux/vim/tmux.vim

```diff
@@ -0,0 +1,2 @@
+" Zoom into tmux test runner pane. To get back to vim, use <C-a><C-p>
+nnoremap <leader>zr :call system("tmux resize-pane -Z -t 1")<cr>
```

Comment:
Assuming `vim-tmux-runner`, this can just be `:VtrFocusRunner<CR>`: https://github.com/gfontenot/dotfiles/blob/master/tag-vim/source/config/tmux.vim#L8